test/test-functions

```diff
@@ -798,7 +798,7 @@ setup_selinux() {
     fi
 
     local conf_dir=/etc/selinux
-    local fixfiles_tools=(awk bash cat chcon expr find grep head secon setfiles rm sort uname uniq)
+    local fixfiles_tools=(awk bash cat chcon expr egrep find grep head secon setfiles rm sort uname uniq)
```

Comment:
Hm? Why egrep?

Reply:
It's, unfortunately, used by `fixfiles` on C8S :(

```
# grep -r egrep /usr/sbin/fixfiles
		grep " $i " /proc/self/mounts | awk '{print $4}' | egrep --silent '(^|,)seclabel(,|$)' && echo $i
		grep " $i " /proc/self/mounts | grep -v "context=" | egrep --silent '(ext[234]| ext4dev | gfs2 | xfs | jfs | btrfs )' && echo $i
	grep " $i " /proc/self/mounts | awk '{print $4}' | egrep --silent '(^|,)rw(,|$)' && echo $i
	grep " $i " /proc/self/mounts | awk '{print $4}' | egrep --silent '(^|,)ro(,|$)' && echo $i
	    egrep -v '(^/home|^/root|^/tmp|^/dev)' |\

```

It was later converted to `grep -E`, but that change is not in C8S (yet).